### ISSUE TITLE
Can retinaface be used instead of sfd?

### ISSUE THREAD

**Issue by Coderx7** [1adrianb/face-alignment#164]
Is it possible to use RetinaFace instead of SFD ?

**Comment by miracleyoo** [comment]
Yes, I succeed in adding RetinaFace into the detector folder and use it instead of SFD. You need to read how the SFD part is written, copy the SFD folder, rename it to retina, and change the __init__.py, do your replacement in the retina_detector.py file... and so on. Just follow how the SFD is implemented and imitate it.

**Comment by 1adrianb** [comment]
@miracleyoo if you have time, feel free to make a pull request :)

**Comment by miracleyoo** [comment]
Sure :) Then I need to clean up the code and figure out how to use torch utils load_url since I put the model in my local space directly now lol.

**Comment by ibadamimarz** [comment]
@miracleyoo Did you get a chance to make a PR for retina ? :) 
Thank you!
